### ISSUE TITLE
Only a batch size of 1 is supported.

### ISSUE THREAD

**Issue by Fuzm7** [orailix/PACT#5]
> Even the way you handled it afterward could easily lead to misunderstandings. Following the stack operation described above, readers might naturally assume the mask is a tensor with shape (bsz, length, 1). However, in reality, it's (1, length, 1) as you pointed out. Only in this case does the subsequent code work, because `squeeze` directly transforms the mask into a one-dimensional tensor, allowing it to be concatenated with `Position_ids`:
>  
>             position_ids = torch.arange(
>                     0, new_input_embeds.shape[1], dtype=torch.long, device=new_input_embeds.device
>                 ).unsqueeze(1).unsqueeze(0)
>             
>             mask=torch.tensor(mask, dtype=torch.bool).squeeze().unsqueeze(1).unsqueeze(0)
> 
> which on line 690 to 694 at [[llava_arch.py.]](https://github.com/orailix/PACT/blob/main/LLaVA-NeXT/llava/model/llava_arch.py) 

 _Originally posted by @Fuzm7 in [#4](https://github.com/orailix/PACT/issues/4#issuecomment-3073860262)_

**Comment by Mohamed-Dhouib** [comment]
I'm closing this  issue as it a duplicate.
